app/src/main/res/drawable/ic_send.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
there should be symmetry in the send/receive icons. I would prefer if this looked more like the receive icon but with the arrow flipped if possible